lib/base/error-handler.js

```diff
@@ -4,122 +4,224 @@
+            "NOT_USE_WITH_OPTIONS" : "Do not use together with options",
+
+            "USE_WITH_OPTIONS" : "Use together with options",
+            "USE_PKGID_PKGINFO" : "packageId must be provided by using either the '--pkgid' or the '--pkginfofile' option",
+            "USE_GDB" : "Please use GDB to debug a native serivce",
+            "NOT_IMPLEMENTED" : "Not implemented",
+            "NOT_HANDLE_FILE" : "Don't know how to handle",
+            "NOT_OVERWRITE_DIR" : "Cannot overwrite the directory",
+
+            "UNMATCHED_DEVICE" : "No matched device",
+            "CANNOT_REMOVE_DEVICE" : "Cannot remove the device",
+            "UNMATCHED_DISPLAY_AFFINITY" : "Please use the same value for \"display\" and \"displayAffinity\"",
+
+            "FAILED_CALL_LUNA" : "luna-send command failed",
+            "FAILED_GET_PORT" : "Failed to get Debug port",
+            "FAILED_GET_SVCPATH" : "Failed to get service installation path",
+            "FAILED_GET_SSHKEY" : "Failed to get ssh private key",
+            "FAILED_TRANSMIT_FILE" : "File transmission error. Please try again",
+            "FAILED_CREATE_DIR" : "Cannot create directory in the destination path - permission denied",
+            "FAILED_REMOVE_PACKAGE" : "No packages installed or removed",
+            "FAILED_FIND_SERVICE" : "Failed to find a service with the specified name",
+
+            "CONNECTED_MULTI_DEVICE" : "Multiple devices are connected by novacom. Please specify the target name",
+            "SET_DEFAULT_MULTI_DEVICE" : "Multiple devices are set to default target. Please reset device list",
+            "NEED_ROOT_PERMISSION" : "Unable to connect to the target device. root access required",
+            "MISSING_CALLBACK" : "Missing completion callback",
+            "USING_WEBINSPECTOR" : "Web Inspector is already connected with another browser. Please close the previous connection",
+            "UNKNOWN_OPERATOR" : "Unknown operator",
+            "UNABLE_USE_SFTP" : "Unable to use sftp",
+        };
+
+    // Pure CLI Tip error : errKey(string), option, value
+    // Luna service error : errKey("FAILED_CALL_LUNA"- string), option, null, heading(service name)
+    // System call & ssh call error : errKey(object)
+    errMsgHdlr.getErrMsg = function(errKey, option, value, heading) {
+        log.info("errMsgHdlr#getErrMsg():", "errKey:", errKey, "option:", option, "value:", value, "heading:", heading);
+        
+        // return errKey when errKey is null
+        if (!errKey) {
+            return errKey;
+        }
+
+        const orgErrKey = errKey,
+            resultErr = [];
+        let mapErrMsg = "";
+
+        // handle systemcall or ssh call err message. errKey is must be object
+        if (typeof(errKey) === "object") {
+            // Change errkey and heading to create Tips meessage;
+            let sysHeading = "syscall";
+
+            // handle as "ssh execution error"
+            if (!orgErrKey.syscall) {
```

Comment:
errkey.syscall

Reply:
I replaced errKey to orgErrKey in handling syscall, service call, lunacall error IF statement.